src/morphocut/image.py

```diff
@@ -326,12 +358,18 @@ class ImageReader(Node):
         fp (file or Variable): A filename (string), pathlib.Path object or file object.
     """
 
-    def __init__(self, fp: RawOrVariable):
+    def __init__(self, fp: RawOrVariable, mode=None):
```

Comment:
Can you rename `mode` to `pil_mode`? Otherwise it might be mistaken for file opening mode (r/w/a/...). Also, this parameter needs to be documented. You can link this section of the pillow documentation: https://pillow.readthedocs.io/en/stable/handbook/concepts.html#modes